src/parser/meta.rs

```diff
@@ -0,0 +1,813 @@
+use nom::{
+    branch::alt,
+    bytes::streaming::{tag, take, take_while1},
+    character::{
+        complete::digit1,
+        streaming::{crlf, space0, space1},
+    },
+    combinator::{map, map_res, value},
+    multi::many0,
+    sequence::{pair, preceded, terminated, tuple},
+    IResult,
+};
+
+use super::{parse_u32, ErrorKind, MetaValue, Response, Status, Value};
+
+// TODO: remove this later in favour of individual parse_meta_*_status methods
+#[allow(dead_code)]
+pub fn parse_meta_status(buf: &[u8]) -> IResult<&[u8], Response> {
+    terminated(
+        alt((
+            value(Response::Status(Status::NotStored), tag(b"NS")),
+            value(Response::Status(Status::Deleted), tag(b"DE")),
+            value(Response::Status(Status::Touched), tag(b"TO")),
+            value(Response::Status(Status::Exists), tag(b"EX")),
+            value(Response::Status(Status::NotFound), tag(b"NF")),
+        )),
+        crlf,
+    )(buf)
+}
```

Comment:
This was an initial approach that will probably be removed in the coming refactor PR, or with the rework of the actual meta_get and meta_set implementations.  It's not possible to use one common status parser because there is overlap / difference in interpretation of status codes across different meta ops.